packages/app/src/app/pages/Dashboard/Components/Repository/ImportRepository.tsx

```diff
@@ -3,8 +3,8 @@ import { useActions } from 'app/overmind';
 import { Stack, Text, Icon, Element } from '@codesandbox/components';
 import css from '@styled-system/css';
 
-export const ImportRepositoryCard: React.FC<{ isDisabled?: boolean }> = ({
-  isDisabled,
+export const ImportRepositoryCard: React.FC<{ disabled?: boolean }> = ({
+  disabled,
```

Comment:
Personally I like `isDisabled`. My rule of thumb would be if it's a boolean and not an html attribute, prefix with `is` or `has`.